### PR TITLE
CASSANDRA-10190: Python 2/3 compatibility for cqlsh

### PR DESCRIPTION
**Pull request by dineshjoshi**

### CODE REVIEW

File: bin/cqlsh.py [hunk 1]
```diff
@@ -140,10 +139,17 @@ def find_zip(libprefix):
     if lib_zip:
         sys.path.insert(0, lib_zip)
 
+import six
+
+from six.moves import configparser
+from six import StringIO
+from six.moves import input
+from six import ensure_text, ensure_str
```
Comment (ptbannister) [inline bin/cqlsh.py]: These imports could be condensed:
`from six import StringIO, ensure_str, ensure_text`
`from six.moves import configparser, input`

File: bin/cqlsh.py [hunk 2]
```diff
@@ -216,8 +222,10 @@ def find_zip(libprefix):
 parser.add_option("-f", "--file", help="Execute commands from FILE, then exit")
 parser.add_option('--debug', action='store_true',
                   help='Show additional debugging information')
-parser.add_option("--encoding", help="Specify a non-default encoding for output." +
-                  " (Default: %s)" % (UTF8,))
+parser.add_option('--coverage', action='store_true',
+                  help='Collect coverage data')
+parser.add_option("--encoding", help="Specify a non-default encoding for output."
+                  + " (Default: %s)" % (UTF8,))
```
Comment (ptbannister) [inline bin/cqlsh.py]: User specified encoding is a great addition.

File: pylib/cqlshlib/test/ansi_colors.py [hunk 3]
```diff
@@ -14,31 +14,36 @@
...
+
 import re
+import six
+
+LIGHT = 0o10
 
-LIGHT = 010
 
-ansi_CSI = '\033['
+#ansi_CSI = '\033['
```
Comment (ptbannister) [inline pylib/cqlshlib/test/ansi_colors.py]: Should remove this commented out line.

**Review by ptbannister — APPROVED** [review]
I commented recommending a couple of minor changes (removing a commented out line, condensing a few imports), but I see no substantive problems. +1.

File: .circleci/config-2_1.yml [hunk 4]
```diff
@@ -138,6 +138,46 @@ j8_with_dtests_jobs: &j8_with_dtests_jobs
     - j8_upgradetests-no-vnodes:
         requires:
           - start_upgrade_tests
+    - start_j8_cqlsh_tests-with-vnodes:
```
Comment (dcapwell) [inline .circleci/config-2_1.yml:141]: I do wish we had a better way to maintain this =(

**Pull request closed without merging**